controllers/statusreport/statusreport_adapter.go

```diff
@@ -142,10 +142,19 @@ func (a *Adapter) EnsureSnapshotFinishedAllTests() (controller.OperationResult,
 		return controller.ContinueProcessing()
 	}
 
-	// Since all integration tests finished, set the Snapshot Integration status as finished, but don't update the resource yet
-	gitops.SetSnapshotIntegrationStatusAsFinished(a.snapshot,
-		"Snapshot integration status condition is finished since all testing pipelines completed")
-	a.logger.LogAuditEvent("Snapshot integration status condition marked as finished, all testing pipelines completed",
+	if len(*integrationTestScenarios) > 0 {
+		// Since all integration tests finished, set the Snapshot Integration status as finished, but don't update the resource yet
+		gitops.SetSnapshotIntegrationStatusAsFinished(a.snapshot,
+			"Snapshot integration status condition is finished since all testing pipelines completed")
+		a.logger.LogAuditEvent("Snapshot integration status condition marked as finished, all testing pipelines completed",
+			a.snapshot, helpers.LogActionUpdate)
+	} else {
+		gitops.SetSnapshotIntegrationStatusAsFinished(a.snapshot,
+			"Snapshot integration status condition is finished since no testing pipelines were run")
+		a.logger.LogAuditEvent("Snapshot integration status condition marked as finished, no integration test scenarios were found",
+			a.snapshot, helpers.LogActionUpdate)
+	}
+	a.logger.LogAuditEvent(fmt.Sprintf("Found %d integration test scenarios", len(*integrationTestScenarios)),
```

Comment:
This is not an audit event, we haven't updated anything here

Reply:
You're right, that was to debug a test issue.  Removed.